### DISCUSSION TITLE
Will uBo also stop working at _some_ point in Thorium, due to Manifest v3 transition?

### DISCUSSION THREAD

**Discussion by convexshiba** [Alex313031/thorium#767] [Q&A] [answered]
titled says it all. Thanks.

**Comment by gz83** [comment, marked as answer]
You can still use UBO with MV2 just fine, and Google hasn't really given up on MV2 yet.

In addition, even if Google completely abandons MV2 in the future, Thorium will have a solution.

**Reply by convexshiba** [reply]
Thanks. Was thinking about what to do if Google completed disallow MV2 exts. 
Good to know that Thorium will have a solution.

**Comment by ScottRFrost** [comment]
FYI - Google removed MV2 from Chrome Canary this week - https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline

Time is running out to get all of your friends migrated over to a browser that will retain support for MV2 extensions
